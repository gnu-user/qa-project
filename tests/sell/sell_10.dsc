TITLE
    Ticket Price Input Format (Non-Decimal Input, garbage)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the sell 
    transaction. Enter a valid title, and enter garbage non-numerical input 
    when prompted to enter a ticket price. Â Verify that the input is not 
    accepted and an error occurs.
DEPENDENCIES
    sell_9
