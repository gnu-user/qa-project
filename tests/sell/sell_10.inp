login
administrator
sell
A valid title.
invaliddollars