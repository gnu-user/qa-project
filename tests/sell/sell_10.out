Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the event title: 
Enter the sale price in dollars: 
Invalid sale price, amount must be in dollars (e.g. 10.00).